### PR TITLE
feat: implement image to image on flux1 model

### PR DESCRIPTION
**Pull request by hugohonda**

### CODE REVIEW

File: vision_agent_tools/models/flux1.py [hunk 1]
```diff
@@ -112,75 +115,89 @@ def __call__(
...
-            raise ValueError(
-                f"`height` and `width` have to be divisible by 8 but are {height} and {width}."
-            )
-
-        if max_sequence_length is not None and max_sequence_length > 512:
-            raise ValueError(
-                f"`max_sequence_length` cannot be greater than 512 but is {max_sequence_length}"
-            )
+        if config.seed is None:
+            config.seed = random.randint(0, 2**32 - 1)
```
Comment (hrnn) [inline vision_agent_tools/models/flux1.py:133]: I'm not sure about modifying the `config`, I think configurations should be immutable.

Comment (hugohonda) [reply vision_agent_tools/models/flux1.py]: Agreed! Fixed, thanks

File: vision_agent_tools/models/flux1.py [hunk 2]
```diff
@@ -287,3 +304,50 @@ def _inpaint_image(
...
+            height (`int`)
+            width (`int`)
+            num_inference_steps (`int`)
+            guidance_scale (`float`)
+            num_images_per_prompt (`int`)
+            generator (`torch.Generator`)
+            max_sequence_length (`int`)
+
+        Returns:
+            Optional[Image.Image]: The generated image if successful; None if an error occurred.
```
[the remark this answers is not in the document]
Comment (hugohonda) [reply vision_agent_tools/models/flux1.py]: You are right, resolved thanks!

**Comment by hugohonda** [comment]
Also updated the docs/flux1.md

**Review by hrnn — APPROVED** [review]
LGTM, :shipit:

**Pull request merged**